modules/partition-replicator/src/integrationTest/java/org/apache/ignite/internal/partition/replicator/utils/TestPlacementDriver.java

```diff
@@ -81,7 +81,11 @@ public CompletableFuture<List<TokenizedAssignments>> getAssignments(
         return failedFuture(new UnsupportedOperationException("getAssignments() is not supported in FakePlacementDriver yet."));
     }
 
-    private CompletableFuture<ReplicaMeta> getPrimaryReplicaMeta() {
+    private CompletableFuture<ReplicaMeta> getPrimaryReplicaMeta(ReplicationGroupId replicationGroupId) {
+        if (replicationGroupId instanceof ZonePartitionId && ((ZonePartitionId) replicationGroupId).zoneId() == 0) {
```

Comment:
Let's name this 0 as a separate DEFAULT_ZONE_ID field

Reply:
Done~